demos/gallery/samples/chart/BubbleXY.js

```diff
@@ -0,0 +1,27 @@
+import { BubbleXY } from "@hpcc-js/chart";
+
+let categories = {};
+let falsey_defaults = {};
+new BubbleXY()._publishedProperties.forEach(prop => {
+    if (!categories[prop.type]) {
+        categories[prop.type] = 0;
+        falsey_defaults[prop.type] = 0;
+    }
+    categories[prop.type]++;
+    if(!prop.defaultValue)falsey_defaults[prop.type]++;
+});
```

Comment:
This code confuses me - so I suspect it would confuse a user, not a great example...

Reply:
I was trying to add an example that uses information from the widget as its own data source. I should probably just add some standard examples.

Reply:
Yup - lets make everything very easy to understand...